streams/src/main/java/org/apache/kafka/streams/processor/internals/AssignedStreamsTasks.java

```diff
@@ -46,6 +45,52 @@ public StreamTask restoringTaskFor(final TopicPartition partition) {
         return restoringByPartition.get(partition);
     }
 
+    @Override
```

Comment:
The following three functions do not have logical changes, just re-grouping all overridden functions on top of AssignedTasks here.